extensions/servlet/src/com/google/inject/servlet/ServletScopes.java

```diff
@@ -218,9 +220,15 @@ public String toString() {
    *     thread, or if the request has completed.
    * 
    * @since 3.0
+   * @deprecated You probably want to use {@code transferRequest} instead
```

Comment:
:(

We currently use this to provide access to `@RequestScoped` objects inside thread pools.  `transferRequest()` doesn't allow parallelism.  I suppose we could use `scopeRequest()` and manually forward a few things like the `HttpServletRequest`.  Is that the recommended way now?